tests/test_geospatial.py

```diff
@@ -0,0 +1,418 @@
+import glob
+import json
+import os
+import shutil
+import tempfile
+import unittest
+import warnings
+from pathlib import Path
+from unittest import mock
+
+import affine
+import fiona
+import numpy as np
+import rasterio
+import shapely
+
+import tiledb
+from tiledb.cloud.geospatial import BoundingBox
+from tiledb.cloud.geospatial import DatasetType
+from tiledb.cloud.geospatial import build_inputs_udf
+from tiledb.cloud.geospatial import ingest_geometry_udf
+from tiledb.cloud.geospatial import ingest_point_cloud_udf
+from tiledb.cloud.geospatial import ingest_raster_udf
+from tiledb.cloud.geospatial import load_geometry_metadata
+from tiledb.cloud.geospatial import load_pointcloud_metadata
+from tiledb.cloud.geospatial import load_raster_metadata
+from tiledb.cloud.geospatial import read_uris
```

Comment:
Likewise with this—importing `tiledb.cloud.geospatial` will end up importing those libraries by proxy, so within the function itself, import the items. To avoid all the repetition, I recommend importing the module rather than anything inside it, so…